README.md

```diff
@@ -52,3 +46,29 @@ curl -LO https://github.com/edenhill/librdkafka/archive/v1.0.1.tar.gz \
   && make \
   && sudo make install
 ```
+## Control Plane
+
+The control plane for the Kafka Channels is managed by the [knative-kafka-controller](./components/controller/README.md)
+which is installed by default into the knative-eventing namespace. `KafkaChannel` custom resource instances can be created in any user
+namespace. The knative-kafka-controller will guarantee that the Data Plane is configured to support the flow of events as
+defined by [Subscriptions](https://knative.dev/docs/reference/eventing/#messaging.knative.dev/v1alpha1.Subscription) to
+a KafkaChannel.  The underlying Kafka infrastructure to be used is defined in a specially
+labeled [secret](./resources/README.md#Credentials) in the knative-eventing namespace.  Knative-Kafka supports several
+different kafka (and kafka-like) [infrastructures](./resources/README.md#Kafka Providers).
+
+## Data Plane
+
+The data plane for all `KafkaChannels` runs in the knative-eventing namespace.  There is a single deployment for the
+receiver side of all channels which accepts CloudEvents and sends them to Kafka.  Each channel uses one kafka topic.
+This deployment supports horizontal scaling with linearly increasing performance characteristics through specifying the number of replicas.
```

Comment:
super minor nit - Kafka should probably be capitalized (it is elsewhere) for consistency?